pages/Nix/Hyprland on Home Manager.md

```diff
@@ -62,3 +71,34 @@ in {
   };
 }
 ```
+{{< /tab >}}
+
+{{< /tabs >}}
+
+
+## Usage
+
+Once the module is enabled, you can use it to declaratively configure Hyprland:
+
+```nix
+# home.nix
+{config, pkgs, ...}: {
+  wayland.windowManager.hyprland.extraConfig = ''
+    % TODO add some examples that add string interpolation or something cool
+  '';
+}
+```
+
+## Plugins
+
+Hyprland plugins can be added through an option:
+
+```nix
+wayland.windowManager.hyprland.plugins = [
+  inputs.hyprland-plugins.packages.${pkgs.system}.hyprbars
+  "/absolute/path/to/plugin.so"
+];
+```
```

Comment:
What about plugins not providing a flake? I think we can give an example of a builder here, which generates a plugin as the output.

Reply:
That'll be in a separate PR. I also want to make a general builder and a flake template, and then I'll also add instructions here.